pkgs/by-name/cj/cjdns/package.nix

```
@@ -63,14 +67,26 @@ rustPlatform.buildRustPackage rec {
     ]
   );
 
+  cargoTestFlags = [
+    # don't run doctests since they fail with "cannot find type `Ctx` in this scope"
+    "--lib"
+    "--bins"
+    "--tests"
+  ];
+
+  checkFlags = [
+    # Tests don't seem to work - "called `Result::unwrap()` on an `Err` value: DecryptErr: NO_SESSION"
+    "--skip=crypto::crypto_auth::tests::test_wireguard_iface_encrypt_decrypt"
+    "--skip=crypto::crypto_auth::tests::test_wireguard_iface_encrypt_decrypt_with_auth"
+  ];
+
   passthru.tests.basic = nixosTests.cjdns;
 
-  meta = with lib; {
-    broken = true; # outdated, incompatible with supported python versions
+  meta = {
     homepage = "https://github.com/cjdelisle/cjdns";
     description = "Encrypted networking for regular people";
-    license = licenses.gpl3Plus;
-    maintainers = with maintainers; [ ehmry ];
-    platforms = platforms.linux;
+    license = lib.licenses.gpl3Plus;
+    maintainers = with lib.maintainers; [ ehmry ];
+    platforms = lib.platforms.linux;
```

this now builds on macos, so let's enable it

```suggestion
    platforms = lib.platforms.linux ++ lib.platforms.darwin;
```

---

Added